Makefile

```diff
@@ -45,6 +45,15 @@ IMAGE_TAG=$(shell ./tools/image-tag)
 
 all: $(SCOPE_EXPORT)
 
+update-cri:
+	curl https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto > runtime/api.proto
+
+protoc-gen-gofast:
+	@go get -u -v github.com/gogo/protobuf/protoc-gen-gofast
+
+cri: update-cri protoc-gen-gofast
```

Comment:
We should only run `make cri` when we want to update the files, at least that was the intention of it, as we don't need to update it every time but when the upstream will get updated, hence the checked in files to guarantee the API does not change each time. That includes the actual `cri` make target which just regenerates the newly downloaded files. Open to suggestions of course!

Reply:
Perfect then!
It can be guessed given the file is checked in, but you may nevertheless want to add a comment next to the `cri` target to clarify this?

Reply:
Makes sense, done! Let me know if it needs rephrasing to understand it better?

Reply:
No, perfect, merge at will! 👍